api/src/main/java/io/opentelemetry/common/Main.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.common;
+
+import java.util.List;
+
+@SuppressWarnings({
+  "BadImport",
+  "UnusedVariable",
+  "PrivateConstructorForUtilityClass",
+  "MultipleTopLevelClasses",
+  "JavadocMethod"
+})
+public class Main {
+
+  public static void main(String[] args) {
+    CleanAttributes attributes =
+        CleanAttributes.newBuilder()
+            .setAttribute("string", "I'm a String")
+            .setAttribute("boolean", true)
+            .setAttribute("double", 33.444d)
+            .setAttribute("long", 34333L)
```

Comment:
I realize this is an aspect of the current API too. Wonder if we need to use `.setLong`, `.setDouble`, etc instead of `.setAttribute` to prevent the easy bug of storing a number as the wrong type.

Reply:
Easy change to make, although a bit orthogonal to this particular bit of work. :)

Reply:
I made the change here, since it was easy to do.